gradle/libs.versions.toml

```diff
@@ -0,0 +1,48 @@
+[versions]
```

Comment:
This file is copied from datamodel-java and necessarily contains a lot of things that can be removed, however, before I do this, I think that it is interesting that nebula doesn't pick up on the fact that the definitions are unused. I guess that unused definitions are not included in gradle's dependency tree